terraform/aws/modules/rds-aurora-postgresql/main.tf

```diff
@@ -73,22 +73,28 @@ resource "aws_rds_cluster" "provisioning_rds_cluster" {
 }
 
 resource "aws_rds_cluster_instance" "provisioning_rds_db_instance" {
-  count                           = var.replica_min
-  identifier                      = format("rds-db-instance-multitenant-%s-%s-%s", split("-", var.vpc_id)[1], local.database_id, (count.index + 1))
-  cluster_identifier              = aws_rds_cluster.provisioning_rds_cluster.id
-  engine                          = var.engine
-  engine_version                  = var.engine_version
-  instance_class                  = var.instance_type
-  db_subnet_group_name            = "mattermost-provisioner-db-${var.vpc_id}-postgresql"
-  db_parameter_group_name         = aws_db_parameter_group.db_parameter_group_postgresql.id
-  preferred_maintenance_window    = var.preferred_maintenance_window
-  apply_immediately               = var.apply_immediately
-  monitoring_role_arn             = data.aws_iam_role.enhanced_monitoring.arn
-  monitoring_interval             = var.monitoring_interval
-  promotion_tier                  = count.index + 1
-  performance_insights_enabled    = local.performance_insights_enabled
+  count                        = var.replica_min
+  identifier                   = format("rds-db-instance-multitenant-%s-%s-%s", split("-", var.vpc_id)[1], local.database_id, (count.index + 1))
+  cluster_identifier           = aws_rds_cluster.provisioning_rds_cluster.id
+  engine                       = var.engine
+  engine_version               = var.engine_version
+  instance_class               = var.instance_type
+  db_subnet_group_name         = "mattermost-provisioner-db-${var.vpc_id}-postgresql"
+  db_parameter_group_name      = aws_db_parameter_group.db_parameter_group_postgresql.id
+  preferred_maintenance_window = var.preferred_maintenance_window
+  apply_immediately            = var.apply_immediately
+  monitoring_role_arn          = data.aws_iam_role.enhanced_monitoring.arn
+  monitoring_interval          = var.monitoring_interval
+  promotion_tier               = count.index + 1
+  performance_insights_enabled = local.performance_insights_enabled
 
-  tags = var.tags
+  tags = merge(
+    {
+      "DatabaseType"                        = "multitenant-rds",
```

Comment:
@stafot please change this DatabaseType tag to be the same as the RDS cluster. It gets the value from a variable now.

Reply:
ack. And has various values?

Reply:
yeap `multitenant-rds` and `multitenant-rds-dbproxy`

Reply:
Hmm, thanks for the info. Need to adjust cost allocation scripts and metrics a bit. 🔥